<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>926</width>
    <height>353</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="Preview" name="PreviewWgt" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>926</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="tearOffEnabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>场景</string>
    </property>
    <addaction name="actionDepthTesting"/>
    <addaction name="actiondepthTestingPrecise"/>
    <addaction name="actionMousePicking"/>
    <addaction name="actionModels"/>
    <addaction name="actionStencilOutLine"/>
    <addaction name="actionDiscard"/>
    <addaction name="actionBlending"/>
    <addaction name="actionCulling"/>
    <addaction name="actionFrameBuffer"/>
    <addaction name="actionPostProcessing"/>
    <addaction name="actionSkyBox"/>
    <addaction name="actionAdvancedData"/>
    <addaction name="actionGLSLVariable"/>
    <addaction name="actionInterfaceBlock"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>定时器</string>
    </property>
    <addaction name="actionTimer"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>模型加载</string>
    </property>
    <addaction name="actionLoadModel"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionTimer">
   <property name="text">
    <string>开始/停止</string>
   </property>
   <property name="shortcut">
    <string>Space</string>
   </property>
  </action>
  <action name="actionDepthTesting">
   <property name="text">
    <string>深度测试</string>
   </property>
   <property name="toolTip">
    <string>深度测试</string>
   </property>
  </action>
  <action name="actionMousePicking">
   <property name="text">
    <string>鼠标拾取</string>
   </property>
  </action>
  <action name="actionLoadModel">
   <property name="text">
    <string>模型加载</string>
   </property>
  </action>
  <action name="actiondepthTestingPrecise">
   <property name="text">
    <string>深度测试精度</string>
   </property>
  </action>
  <action name="actionModels">
   <property name="text">
    <string>模型场景+旋转位移</string>
   </property>
  </action>
  <action name="actionStencilOutLine">
   <property name="text">
    <string>模板测试(边框)</string>
   </property>
  </action>
  <action name="actionDiscard">
   <property name="text">
    <string>丢弃片段</string>
   </property>
  </action>
  <action name="actionBlending">
   <property name="text">
    <string>混合模式</string>
   </property>
  </action>
  <action name="actionCulling">
   <property name="text">
    <string>面剔除</string>
   </property>
  </action>
  <action name="actionFrameBuffer">
   <property name="text">
    <string>帧缓冲</string>
   </property>
  </action>
  <action name="actionPostProcessing">
   <property name="text">
    <string>后处理</string>
   </property>
  </action>
  <action name="actionSkyBox">
   <property name="text">
    <string>天空盒子</string>
   </property>
  </action>
  <action name="actionAdvancedData">
   <property name="text">
    <string>高级数据(VBO操作)</string>
   </property>
  </action>
  <action name="actionGLSLVariable">
   <property name="text">
    <string>GLSL变量</string>
   </property>
  </action>
  <action name="actionInterfaceBlock">
   <property name="text">
    <string>接口块</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Preview</class>
   <extends>QWidget</extends>
   <header location="global">preview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
